### ISSUE TITLE
Clippings/wx: Unable to copy text from page inside HTML frame

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#83]
Clippings/wx 6.0pre1 on Google Chrome

Unable to copy text from a web page inside an HTML frame, whether the text is from a textbox or is the body text of the web page.  The issue has been narrowed down to content.js: `handleRequestNewClipping()` - need to investigate further.

The current workaround is to use the `selectionText` property on the event info object passed to the onClicked event listener for the context menu, in background.js. However, this returns text with line breaks collapsed, which is not suitable for saving as a clipping.

**Comment by aecreations** [comment]
Noticed in the content script that the handler of the "ae-clippings-new" message that was sent from the background script is being called multiple times; see copy-and-paste of debug log output.  This only happens if the page is inside an HTML frame.
```
Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLFrameElement]
content.js:182 Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLBodyElement]
content.js:54 Content from HTMLBodyElement: 
content.js:182 Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLBodyElement]
content.js:54 Content from HTMLBodyElement: An input form where text must be entered
content.js:182 Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLBodyElement]
content.js:54 Content from HTMLBodyElement: 
content.js:182 Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLBodyElement]
content.js:54 Content from HTMLBodyElement: 
content.js:182 Clippings/wx: Content script received message 'ae-clippings-new' from extension
content.js:32 Clippings/wx: content.js::handleRequestNewClipping(): activeElt = [object HTMLBodyElement]
content.js:54 Content from HTMLBodyElement:

**Comment by aecreations** [comment]
Issue still happening when attempting to create a clipping from selected text in Gmail's message compose window.

**Comment by aecreations** [comment]
Fixed in Clippings/wx 6.0 alpha 1
